jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -225,10 +243,28 @@ pipeline {
                             }
                             postCleanup()
                         }
-                        failure {
+                        aborted {
+                            script {
+                                def stackName = "test-single-${BUILD_ID}-${ARCHITECTURE}-perf-test"
+                                withCredentials([string(credentialsId: 'jenkins-aws-account-public', variable: 'AWS_ACCOUNT_PUBLIC')]) {
+                                    withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
+                                        try {
+                                            def stack = cfnDescribe(stack: stackName)
+                                            if (stack != null) {
+                                                cfnDelete(stack: stackName, pollInterval:1000)
```

Comment:
nit: Adding logging saying `Deleting stack ${stackName}`

Reply:
indeed ...we can provide more useful input in the output for possible troubleshooting ... let me update it .. thanks!